db/migrate/20211227020940_create_user_albums.rb

```diff
@@ -1,9 +0,0 @@
-class CreateUserAlbums < ActiveRecord::Migration[6.1]
```

Comment:
one more time, you cannot delete any existing migrations, if you want to remove table create new migration.